circuits/core/events.py

```diff
@@ -136,7 +136,7 @@ def __repr__(self):
 
         data = "%s %s" % (
             ", ".join(repr(arg) for arg in self.args),
-            ", ".join("%s=%s" % (k, repr(v)) for k, v in self.kwargs.items())
+            ", ".join("%s=%r" % (k, v) for k, v in self.kwargs.items())
```

Comment:
@spaceone I guess from a readability perspective I find ``str.format()`` easier on the eyes when reading code as well as ``{0}`` easier to read vs. ``%s``. I have actually developed a habit of converting any ``"foo %s" % "bar"`` to ``"foo {0}".format("bar")`` so if I merge this in; it'll likely get changed! Happy to incorporate the other fixes!